src/main/java/interpol/entity/Crime.java

```diff
@@ -17,6 +19,11 @@ public class Crime {
     @Column (name = "punishment")
     private String punishment;
 
+    // Переделанная связь ManyToMany между OrderDetail и Crime на 2 связи OneToMany между OrderDetail, Crime
+    // и промежуточной таблицей CriminalCrimeMap
+    @OneToMany(fetch = FetchType.EAGER, mappedBy = "crime")
+    private Set<Crime> crimes;
```

Comment:
Actually, List is preferable in such kind of relations, let's replace it here and in other classes

Reply:
Also, why `crimes`? It seems, it should be CriminalCrimeMap?